Makefile

```diff
@@ -13,49 +13,59 @@
 # limitations under the License.
 
 PKG=sigs.k8s.io/ibm-powervs-block-csi-driver
-IMAGE?=quay.io/powercloud/ibm-powervs-block-csi-driver
-VERSION=v0.0.1
-GIT_COMMIT?=$(shell git rev-parse HEAD)
+GIT_COMMIT?=$(shell git rev-parse --short HEAD)
 BUILD_DATE?=$(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
-LDFLAGS?="-X ${PKG}/pkg/driver.driverVersion=${VERSION} -X ${PKG}/pkg/driver.gitCommit=${GIT_COMMIT} -X ${PKG}/pkg/driver.buildDate=${BUILD_DATE} -s -w"
+IMAGE?=gcr.io/k8s-staging-cloud-provider-ibm/ibm-powervs-block-csi-driver
```

Comment:
we should have REGISTRY and IMAGE both
something like this:

```
REGISTRY?=gcr.io/k8s-staging-cloud-provider-ibm
IMAGE=ibm-powervs-block-csi-driver
```

Reply:
later you can use `$(REGISTRY)/$(IMAGE)` needed.

Reply:
fixed